core/templates/dev/head/pages/exploration_player/CurrentInteractionService.js

```diff
@@ -42,39 +38,37 @@ oppia.factory('CurrentInteractionService', [
       },
       registerCurrentInteraction: function(submitAnswer, validityCheckFn) {
         /**
-         * Every interaction directive should register themselves here,
-         * even those which do not use the progress nav. All arguments are
-         * optional.
+         * Each interaction directive should call registerCurrentInteraction
+         * when the interaction directive is first created.
          *
-         * @param {function} submitAnswer - Should grab the learner's answer
-         *   and pass it to onSubmit. The interaction can pass in null if
-         *   it does not use the progress nav's submit button
+         * @param {function|null} submitAnswer - Should grab the learner's
+         *   answer and pass it to onSubmit. The interaction can pass in
+         *   null if it does not use the progress nav's submit button
          *   (ex: MultipleChoiceInput).
          * @param {function} validityCheckFn - The progress nav will use this
-         *   to decide whether or not to disable the submit button. The
-         *   interaction can pass in null if it does not use the progress
-         *   nav's submit button.
+         *   to decide whether or not to disable the submit button. If the
+         *   interaction passes in null, the submit button will remain
+         *   enabled (for the entire duration of the current interaction).
          */
-        _submitAnswerFn = submitAnswer;
-        _validityCheckFn = validityCheckFn || _defaultValidityCheck;
+        _submitAnswerFn = submitAnswer || null;
```

Comment:
Nit: the argname should be submitAnswerFn, for parity with validityCheckFn.

Reply:
Done